pkg/listener/kafka/kafka-consumer.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"os"
+	"os/signal"
+	"strconv"
+	"strings"
+
+	"github.com/Shopify/sarama"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	brokers := os.Getenv("BROKER_LIST")
+	if brokers == "" {
+		logrus.Fatalf("broker can't be empty. Please set env BROKER_LIST.")
+	}
+	brokerList := strings.Split(brokers, ",")
+	topic := os.Getenv("TOPIC")
+	if topic == "" {
+		logrus.Fatalf("topic can't be empty. Please set env TOPIC.")
+	}
+	partition := os.Getenv("PARTITION")
+	if partition == "" {
+		partition = "0"
+	}
+
+	config := sarama.NewConfig()
+	config.Consumer.Return.Errors = true
+	master, err := sarama.NewConsumer(brokerList, config)
+	if err != nil {
+		logrus.Fatal(err)
+	}
+	defer func() {
+		if err := master.Close(); err != nil {
+			logrus.Fatal(err)
+		}
+	}()
+	p, err := strconv.Atoi(partition)
+	if err != nil {
+		logrus.Fatal(err)
+	}
+	consumer, err := master.ConsumePartition(topic, int32(p), sarama.OffsetOldest)
```

Comment:
we should probably add the [consumer to a group](https://github.com/Shopify/sarama/blob/08ccbbbceb43303cfdbc076abe55c879842e0e8e/broker.go#L299) (depending on the function ID). A consumer group is a relation between several consumers with a topic. Only one consumer in a group will receive a message. This will be used to replicate the consumer if needed.